_pkgdown.yml

```diff
@@ -20,17 +20,8 @@ development:
 authors:
   Sam Abbott:
     href: "https://www.samabbott.co.uk/"
-    
-navbar:
-  structure:
-    left: [reference, articles, news]
-    right: [github]
```

Comment:
to confirm going to the default, which will be
```
navbar:
  structure:
    left: [intro, reference, articles, tutorials, news]
    right: [search, github]
```
left: no actual tutorials, but would auto-appear if added; otherwise same, except getting started article moved to intro
right: adding search

Reply:
yes I think so. Looks good to me.